sdks/python/apache_beam/pipeline.py

```diff
@@ -28,15 +28,18 @@
 Typical usage:
 
   # Create a pipeline object using a local runner for execution.
-  pipeline = Pipeline(runner=DirectPipelineRunner())
+  p = beam.Pipeline('DirectPipelineRunner')
 
   # Add to the pipeline a "Create" transform. When executed this
   # transform will produce a PCollection object with the specified values.
-  pcoll = pipeline.create('label', [1, 2, 3])
+  c = p | 'label' >> beam.Create([1, 2, 3])
```

Comment:
You can omit the label or give at a more meaningful label. Either one of the things would be fine:
```
pcoll = p | 'create' >> beam.Create([1, 2, 3])
pcoll = p | beam.Create([1, 2, 3])
```
I think I prefer `pcoll` over `c` as a variable name. We use `p` for pipeline in general but for other things we have more explicit names.